src/ripple/protocol/jss.h

```diff
@@ -193,6 +195,8 @@ JSS(converge_time_s);        // out: NetworkOPs
 JSS(cookie);                 // out: NetworkOPs
 JSS(count);                  // in: AccountTx*, ValidatorList
 JSS(counters);               // in/out: retrieve counters
+JSS(cur_a);                  // out: BookChanges
+JSS(cur_b);                  // out: BookChanges
```

Comment:
could we come up with a better name for this? I'd spell out `currency` at the very least. What determines which currency is `a` versus which currency is `b`?